packages/eslint-config/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@agoric/eslint-config",
+  "version": "0.1.0",
+  "description": "Agoric style rules",
+  "private": true,
```

Comment:
Why private?

Reply:
I’m going to duplicate this in `ses` and converge them, then probably publish from `ses`.